test/integration/execute.spec.ts

```diff
@@ -1,32 +1,34 @@
+import { strict as assert } from 'assert';
 import { ActionLibrary } from '@balena/jellyfish-action-library';
+import {
+	cardMixins,
+	Kernel,
+	testUtils as coreTestUtils,
+} from '@balena/jellyfish-core';
 import { DefaultPlugin } from '@balena/jellyfish-plugin-default';
 import { ProductOsPlugin } from '@balena/jellyfish-plugin-product-os';
-import { integrationHelpers } from '@balena/jellyfish-test-harness';
-import { core } from '@balena/jellyfish-types';
-import { TriggeredActionContract } from '@balena/jellyfish-types/build/worker';
-import { strict as assert } from 'assert';
-import * as _ from 'lodash';
-import { Worker } from '../../lib';
+import type { ActionRequestContract } from '@balena/jellyfish-types/build/core';
+import type { TriggeredActionContract } from '@balena/jellyfish-types/build/worker';
+import _ from 'lodash';
+import { testUtils as workerTestUtils } from '../../lib';
```

Comment:
Not only on this file but in general I'm against this kind of renaming. The symbols in our package are more important than those coming from anywhere else and we shouldn't make way for them. It's `testUtils` because it's our testing utilities. Core's testing utilities is `coreTestUtils` because it's not ours and it conflicts with our `testUtils`.

And in general, renaming internal symbols just, IMO, lead to confusion. Is `workerTestUtils` our `testUtils` or something else?